src/compiler/diagnosticMessages.json

```
@@ -3361,6 +3361,11 @@
         "category": "Error",
         "code": 2836
     },
+    "Variable '{0}' is declared in a prior case block.": {

```

case needs to be quoted

```suggestion
    "Variable '{0}' is declared in a prior 'case' block.": {

```

It's tough to give some more context, but I think it could be helpful. Maybe something like

```suggestion
    "Variable '{0}' is declared in a prior 'case' block and cannot be referenced if that declaration does not execute.": {
```